src/main/java/james/altoclef/taskcreator/graphics/AltoFrame.java

```diff
@@ -399,6 +399,23 @@ public void actionPerformed(ActionEvent e) {
 
                 }
             });
+            fileMenu_subMenuOpen_fromFilePath.addActionListener(new ActionListener() {
+                @Override
+                public void actionPerformed(ActionEvent actionEvent) {
+                    PathImporter importer = new PathImporter();
+                    importer.setDefaultCloseOperation(WindowConstants.DISPOSE_ON_CLOSE);
+                    importer.setTitle("Path importer");
+                    importer.setSize(450, 200);
+                    importer.setLocationRelativeTo(null);
+                    importer.setVisible(true);
```

Comment:
Since the new class is used only a singular time, it is best to move these items to the constructor, which makes `AltoFrame` cleaner. 

Move these lines into the default constructor for the dialogue (shown below)